packages/agoric-cli/test/inter-cli.test.js

```diff
@@ -617,12 +611,9 @@ first bid update:
 {"id":"bid-1680212903989","price":"0.81 IST/ATOM","give":{"Bid":"0.5IST"},"want":"3ATOM","result":"Your bid has been accepted"}
 */
 
-test.todo('execSwingsetTransaction returns non-0 code');
-
-test.todo('already cancelled bid');
 /*
 $ agops inter bid cancel --from gov2 bid-1680211556497
 bid-1680211556497 not in live offer ids: bid-1680211593489,bid-1680212903989,bid-1680213097499,bid-1680220217218,bid-1680220368714,bid-1680220406939
 */
 
-test.todo('--give without number');
+// TODO improve test coverage https://github.com/Agoric/agoric-sdk/issues/9965
```

Comment:
I think we should leave this file alone so that it continues to generate warnings until #9965 is resolved (i.e., the warnings are legitimate).

Reply:
My position is that the warnings already did their job to get our attention resulting in $9965 being filed. Now the work can actually be sized up, prioritized, scheduled, tracked, and hopefully completed via #9965. There is nothing else that these warnings can add to #9965 by being there for every lint run. An argument can be made that "well-known warnings" only detract valuable resource by having everyone look at them only to realize that we've already seen and acted upon them, and then simply ignore.